### DISCUSSION TITLE
Archive Keyboard Shortcut

### DISCUSSION THREAD

**Discussion by ColemanRodriguez** [immich-app/immich#6069] [Feature Request]
I'm diving headfirst into organizing my 55,000-photo library, and let me tell you, 20,000+ photos destined for the archive is no small feat! While the app's existing functionalities are awesome, one missing piece could massively boost my efficiency and preserve my sanity would be a keyboard shortcut for archiving photos. If this could added I would be eternally grateful.

**Comment by martabal** [comment]
It's already implemented, just press `Shift + A`
